transport-classes-epoll/src/main/java/io/netty/channel/epoll/EpollEventLoop.java

```diff
@@ -347,8 +347,11 @@ protected void run() {
                                     strategy = epollWaitNoTimerChange();
                                 } else {
                                     // Timerfd needs to be re-armed or disarmed
-                                    prevDeadlineNanos = curDeadlineNanos;
-                                    strategy = epollWait(curDeadlineNanos);
+                                    long result = epollWait(curDeadlineNanos);
+                                    // The result contains the actual return value and if a timer was used or not.
+                                    // We need to "unpack" it. Be aware that the same
```

Comment:
the same what? :)

Reply:
I've missed it...and now I'm curious as well :D

Reply:
fixed :D